src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -11925,5 +11925,26 @@ public class C(int x) : Base(x)
                 Await state.AssertCompletionItemsContainAll("X")
             End Using
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub TestItemsSorted()
```

Comment:
import completion is disabled in the test by default. we'd like to turn it on for validation here.